plugin/src/globals.ts

```diff
@@ -111,28 +111,6 @@ export const defaultGlobals = new Set([
 
   // Hermes
   'HermesInternal',
-
-  // Reanimated
-  '_WORKLET',
-  '_IS_FABRIC',
-  '_log',
-  '_toString',
-  '_scheduleOnJS',
-  '_scheduleOnRuntime',
-  '_makeShareableClone',
-  '_updatePropsPaper',
-  '_updatePropsFabric',
-  '_removeFromPropsRegistry',
-  '_measurePaper',
-  '_measureFabric',
-  '_scrollToPaper',
-  '_dispatchCommandPaper',
-  '_dispatchCommandFabric',
-  '_setGestureState',
-  '_notifyAboutProgress',
-  '_notifyAboutEnd',
-  '_runOnUIQueue',
-  '_getAnimationTimestamp',
```

Comment:
This is potentially a breaking change if someone uses some of those identifiers in worklets. Let's move on to using `global` in Reanimated but keep this list for now.